skopt/gp_opt.py

```diff
@@ -95,7 +96,9 @@ def gp_minimize(func, dimensions, base_estimator=None, acq="LCB", xi=0.01,
         Number of calls to `func`.
         If `n_random_starts` > 0, `n_calls - n_random_starts`
         additional evaluations of `func` are made that are guided
-        by the `base_estimator`.
+        by the `base_estimator`. If `x0` is provided but not `y0`
+        then `n_calls - len(x0) - n_random_starts` evaluations
+        are made instead of `n_calls - n_random_starts` .
```

Comment:
Move the general discussion about the interplay of n_calls and n_random_starts to above the section `Parameters`. Keep the documentation of the parameter `n_calls` where it is :) This would be my proposal.